build.cmd

```diff
@@ -6,13 +6,15 @@ set __BuildArch=x64
 set __BuildType=debug
 
 :: Set the various build properties here so that CMake and MSBuild can pick them up
-set __ProjectDir=%~dp0
-set __ProjectFilesDir=%~dp0
-set __SourceDir=%__ProjectDir%\src
-set __PackagesDir=%__SourceDir%\.nuget
-set __RootBinDir=%__ProjectDir%\binaries
-set __LogsDir=%__RootBinDir%\Logs
-set __CMakeSlnDir=%__RootBinDir%\CMake
+set "__ProjectDir=%~dp0"
```

Comment:
No it's correct. Otherwise we get the quotes inside the environment variable which wreaks havoc if we try to quote it again later.